be2-scala/src/main/scala/ch/epfl/pop/storage/DbActor.scala

```diff
@@ -139,8 +139,8 @@ final case class DbActor(
   }
 
   @throws[DbActorNAckException]
-  private def createElectionData(electionId: Hash, keyPair: KeyPair): Unit = {
-    val channel = Channel(s"${ROOT_CHANNEL_PREFIX}private/${electionId.toString}")
+  private def createElectionData(electionId: Hash, keyPair: KeyPair, laoId: Hash): Unit = {
```

Comment:
Here and elsewhere, try to define function signatures where the parameter order is obvious to someone calling. Because there's a relationship between `LAO` and `election`, they should at least be next to each other, and it might even make sense to ask `laoId` first considering their hierarchical relationship.